notebooks/lightning_segmentation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>typo:<code> serves as an educational example</code></p><p> </p><p>Can we link to the future classification notebook to show the already build in affine transformations? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Linking could be possible, but not that easy, if we want our tutorials link to tutorial section and our nb link to the gh nb.